docs/README.md

```diff
@@ -0,0 +1,29 @@
+# Fortuna Documentation
+
+The documentation for Fortuna can be found [here](https://aws-fortuna.readthedocs.io/en/latest/).
+
+## Build the documentation
+
+The documentation for Fortuna is comprised of a series of notebooks. To serve these locally, there are two steps that need to be taken that we outline below.
+
+### Prerequisites
+
+To build the documentation, first install Fortuna and it's dependencies by following the [installation instructions](https://github.com/awslabs/fortuna#installation). Next, install the documentation requirements through
+```bash
+pip install -r docs/requirements.txt
```

Comment:
I think this should change. We should try to do everything through poetry.

Reply:
Agreed.